cellassign_run_predictor.R

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env Rscript 
```

Comment:
This script should be named for the function it wraps- i.e. cellassign-cellassign.R